pages/07.help/help.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Help me
+template: docs
+taxonomy:
+    category: docs
+routes:
+  default: '/help-me'
+---
+
+Even though YunoHost contributors try to continuously perfect it, YunoHost itself or this documentation are not free from bugs or flaw. To let us help you, however, you need to follow these guidelines.
+
+Support is only available on [the forum](https://forum.yunohost.org?target=_blank) or our [chatrooms](/chat_rooms?target=_blank).
+We are also on Mastodon, but we do not offer help on this platform.
+
+# Dos
+
+## Be courteous and patient
+Help in the chatrooms or on the forum is given by our users and contributors, all volunteers. You may be in a hurry, you may be exhausted by fruitless trials... Have a break, have a sip of your preferred soothing beverage, then continue reading. We need you in a good mindset to help you.
+
+## Explain the context
+Your request needs to be stated in a complete but concise way. Explain plainly what you want to achieve and in what manner it is not working for you.
+
+## Show errors and logs
+This is absolutely paramount if YunoHost or a command fails. It always explains why, through a single error code or a full wall of seemingly illegible text.
+
+For the latter, YunoHost provides green "Share with YunoPaste" buttons when something fails or in the Services page of the webadmin. Click on it, it will open a new web page, and you can then share its URL.
+
+![YunoPaste button](image://yunopaste.png)
+
+![YunoPaste from install error](image://yunopaste_install.mp4?loop=1&controls=0&autoplay=1&muted)
+![YunoPaste from service](image://yunopaste_service.mp4?loop=1&controls=0&autoplay=1&muted)
+
+If your issue is not related to an installation or upgrade process but instead is about an app failing load properly, ideally you can investigate with your browser console (opened with the F12 key): its Network tab can show you HTTP error codes (including the infamous 403, 404, 500...), and its Console can output errors too.
+
+Screenshots are most welcome.
+
+To guide you, on the forum you need to fill in the Support request template.
+
+# Don'ts
```

Comment:
```suggestion
# Dont's
```

Reply:
I respectfully disagree: https://dictionary.cambridge.org/dictionary/english/dos-and-don-ts 😛

Reply:
Ooooh alrighty sorry haha